### ISSUE TITLE
i have some questions on the processing of datasets

### ISSUE THREAD

**Issue by shaoyuyoung** [antonio-mastropaolo/ICSME2021-Completion#6]
hello,in your code,you use the "<extra_id_0>" as a PLACEHOLDER_MASK.I wonder if I can replace it with something else,for example,"<mask>" without modifying other code.Moveover,using differnet PLACEHOLDER_MASK  whether have a significant impact on the prediction of the model?
I am a noob in DL,I am very interested in your research.SO these quetions seems silly.Thank you very much.

**Comment by antonio-mastropaolo** [comment]
Hello!

If you are talking about the pre-training phase, then you should have several <extra_id_xx> tokens for the masked part.
It might happen that if the comment that we mask is featured just by few tokens, then only one <extra_id_xx> token will take the place of the raw token.

As for fine-tuning, only one <extra_id> token is needed seen the way we instantiate the model for the task of code comment completion.

In general, the <extra_id_xx> token has a specific meaning and is a reserved token for the model. Thus, if you replace the <extra_id> with double quotes, the model does not know anymore what it has to be done.
On a similar note, I think you can define your own PLACEHOLDERS and replace the <extra_id_xx> with these, but the tokenizer must be re-trained to include your EXTRA PLACEHOLDERS.
